bigtop-deploy/puppet/hieradata/bigtop/cluster.yaml

```diff
@@ -192,6 +192,9 @@ hadoop_hive::common_config::hbase_zookeeper_quorum: "%{hiera('hadoop_hbase::comm
 hadoop_hive::common_config::kerberos_realm: "%{hiera('kerberos::site::realm')}"
 hadoop_hive::common_config::metastore_uris: "thrift://%{hiera('bigtop::hadoop_head_node')}:9083"
 
+hadoop_hive::common_config::hive_zookeeper_quorum: "%{hiera('hadoop_hive::common_config::hive_zookeeper_quorum')}"
```

Comment:
Per the dev list [discussion](https://mail-archives.apache.org/mod_mbox/bigtop-dev/201803.mbox/%3C85DF9170-CEC9-4462-B554-5CAF892B0570%40oflebbe.de%3E), we don't need to alter cluster.yaml for these new properties.

Reply:
Doh!  @jamesbeedy you ninja'd me :)  Thx for removing `cluster.yaml` updates.